apps/web/app/api/escrow/initialize/route.ts

```
@@ -0,0 +1,105 @@
+import { NextRequest, NextResponse } from "next/server";
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)